src/components/header.tsx

```diff
@@ -0,0 +1,24 @@
+import { AppBar } from "@mui/material";
+import Toolbar from "@mui/material/Toolbar";
+import { ContainedButton, OutlinedButton } from "./buttons";
+
+const Header = () => {
+  return (
+    <>
+      <div>Header</div>
+      <AppBar  sx={{ backgroundColor: "white" ,display:'flex',alignItems:"flex-end"}}>
```

Comment:
Using "sx" is fine in a small application but it is not scalable. Try refactoring it with the "Emotion" library. Even the MUI docs suggest using Emotion to handle our styles.